catfruits.go

```diff
@@ -2,30 +2,42 @@ package catfruits
 
 // Infomation is a repository infomation.
 type Infomation struct {
-	Frameworks []string `json:"frameworks"`
+	Frameworks []string            `json:"frameworks"`
+	Packages   map[string][]string `json:"packages"`
 }
 
 // Scanner is scanner for repository.
 type Scanner struct {
 	dir    string
 	fwFunc map[string]FrameworkFunc
+	info   *Infomation
 }
 
 // FrameworkFunc is a function that detects framework
 type FrameworkFunc func(*Scanner) (ok bool, err error)
+type PackageFunc func(*Scanner) (pkgs []string, err error)
```

Comment:
__[gometalinter]__
* [golint] exported type PackageFunc should have comment or be unexported